code/game/objects/items/storage/boxes/security_boxes.dm

```diff
@@ -225,8 +225,8 @@
 	illustration = "sandbag"
 
 /obj/item/storage/box/emptysandbags/PopulateContents()
-	for(var/i in 1 to 7)
-		new /obj/item/emptysandbag(src)
+	for(var/i in 1 to 2)
+		new /obj/item/stack/sheet/mineral/emptysandbags/fifty(src)
```

Comment:
wayyyy more, intentionally? why?

Reply:
The only reason for that is because #1, I wanted to have more than one stack in a box or else it would look kind of awkward, but I still wanted the max amount per stack to be 50, #2 I don't think it matters at all since I have never seen them used literally ever since I added them